device/tt_cluster_descriptor.cpp

```diff
@@ -297,7 +299,7 @@ std::unique_ptr<tt_ClusterDescriptor> tt_ClusterDescriptor::create_from_yaml(con
 
     std::ifstream fdesc(cluster_descriptor_file_path);
     if (fdesc.fail()) {
-        throw std::runtime_error("Error: cluster connectivity descriptor file " + cluster_descriptor_file_path + " does not exist!");
+        throw std::runtime_error(fmt::format("Error: cluster connectivity descriptor file {} does not exist!", cluster_descriptor_file_path));
```

Comment:
I wonder if it makes sense for us to have a macro like `UMD_THROW` that throws a `std::exception`-derived UMD exception type and that applies formatting:

```
UMD_THROW("Cluster file {} doesn't exist!", path);
```

Reply:
Yeah it makes sense, we have an issue for that - https://github.com/tenstorrent/tt-umd/issues/55. It will be adressed in separate PR